src/core/default/mappers/simple/dynamodb_deployer.py

```diff
@@ -91,6 +91,7 @@ def _remove_simple_dynamodb_table(
     )
 
 
+# ANIBAL return type doesn't match
```

Comment:
This should also be Option[Dict]. Like the `lambda` mapper function, this returns the Dict from the _create and _update functions then None when deleting the resource